packages/rum/src/domain/rumEventsCollection/view/trackViews.ts

```diff
@@ -83,6 +97,11 @@ export function trackViews(location: Location, lifeCycle: LifeCycle) {
   )
 
   return {
+    addTiming(name: string, inInitialView = false) {
+      const view = inInitialView ? initialView : currentView
+      view.addTiming(name, performance.now())
```

Comment:
```suggestion
      view.addTiming(name)
```
no need for a time parameter then